### PR TITLE
Feature/UuiDropdownMenu

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: app/src/docs/_examples/tables/TablesExamples.module.scss [hunk 1]
```diff
@@ -30,3 +30,11 @@
 .weekend-day {
     background-color: #EBEDF5;
 }
+
+.action-button {
+    padding: 0 20px;
```
Comment (AlekseyManetov) [inline app/src/docs/_examples/tables/TablesExamples.module.scss:35]: why we need to add custom styles here? I supposed that we should use DropdownMenuButton without any additional customization

File: loveship/components/dev/MakeMeItem.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { DropdownMenuButton } from '../overlays';
+import { DropdownMenuButton } from '../../components';
```
Comment (AlekseyManetov) [inline loveship/components/dev/MakeMeItem.tsx:2]: such import will cause cycle dependencies, just make direct export from @epam/uui

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: app/src/docs/_examples/tables/TablesExamples.module.scss (hunk 1 above)
Comment (vik753) [reply app/src/docs/_examples/tables/TablesExamples.module.scss:35]: removed

File: loveship/components/dev/MakeMeItem.tsx (hunk 2 above)
Comment (vik753) [reply loveship/components/dev/MakeMeItem.tsx:2]: done

Comment (vik753) [reply loveship/components/dev/MakeMeItem.tsx:2]: Refactored DropdownMenu props: 
interface IDropdownMenuBody extends DropdownBodyProps, IHasCX, IHasChildren, IHasRawProps<HTMLAttributes<HTMLDivElement>>, IHasForwardedRef<HTMLDivElement> {
    minWidth?: number;
    closeOnKey?: React.KeyboardEvent<HTMLElement>['key'];
}
and now we can setup minWidth for container like that:  <DropdownMenuBody minWidth={ 90 } { ...dropdownProps }>

**Comment by vik753** [comment]
Refactored DropdownMenu props:
interface IDropdownMenuBody extends DropdownBodyProps, IHasCX, IHasChildren, IHasRawProps<HTMLAttributes>, IHasForwardedRef {
minWidth?: number;
closeOnKey?: React.KeyboardEvent['key'];
}
and now we can setup minWidth for container like that: <DropdownMenuBody minWidth={ 90 } { ...dropdownProps }>

**Pull request merged**